unit-tests/func/extrinsics/test-extrinsics.cpp

```diff
@@ -0,0 +1,205 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2021 Intel Corporation. All Rights Reserved.
+
+/////////////////////////////////////////////////////////////////////////////
+// This set of tests is valid for any device that supports the HDR feature //
+/////////////////////////////////////////////////////////////////////////////
+
+#define CATCH_CONFIG_MAIN
+#include "../../catch.h"
+#include "../func-common.h"
+#include <librealsense2/rsutil.h>
+
+using namespace rs2;
+
+// structure of a matrix 4 X 4, representing rotation and translation as following:
+// pos_and_rot[i][j] is 
+//  _                        _ 
+// |           |              |
+// | rotation  | translation  |
+// |   (3x3)   |    (3x1)     |
+// | _________ |____________  |
+// |     0     |      1       |
+// |_  (1x3)   |    (1x1)    _|
+//
+struct position_and_rotation {
+    double pos_and_rot[4][4];
+    // rotation tolerance - units are in cosinus of radians
+    const double rotation_tolerance = 0.000002;
+    // translation tolerance - units are in meters
+    const double translation_tolerance = 0.0021; // 2.1mm
```

Comment:
To be verified - the envisaged tolerance is <1mm

Reply:
max tolerance of 1.05mm achieved